core/sys/win32/windows.odin

```diff
@@ -19,12 +20,14 @@ Hgdiobj   :: distinct Handle;
 Hmodule   :: distinct Handle;
 Hmonitor  :: distinct Handle;
 Hrawinput :: distinct Handle;
+Hresult   :: distinct int;
 HKL       :: distinct Handle;
 Wparam    :: distinct uint;
 Lparam    :: distinct int;
+Lpvoid    :: distinct rawptr;
```

Comment:
It's better to just use a plain `rawptr` rather than having a distinct type for `Lpvoid`.